### DISCUSSION TITLE
Is there a reason to not see many charts with drawn dependent axis?

### DISCUSSION THREAD

**Discussion by SasheVuchkov** [observablehq/plot#1977] [Q&A]
Hi,

I'm trying to create some charts with rules on axes drawn in different colors and thicknesses. I'm experiencing difficulties drawing a rule for the dependent axis with `Plot.ruleY()`. It just doesn't show up anywhere. 

Am I missing anything?

**Comment by Fil** [comment]
Please share some code and maybe a drawing of what's missing? it's kinda hard to guess.
